docs/dev/plugins/Debugging.md

```diff
@@ -0,0 +1,11 @@
+# Debugging a Mumble plugin
+
+Generally you have to resort to the standard debugging techniques when it comes to debugging the functionality inside your plugin. If however you want
+to find out why Mumble doesn't recognize any given function in your plugin or you think that the issue may lie within Mumble somewhere, you have to
+[build Mumble](../build-instructions/README.md) yourself.
+
+When doing so, you activate the `plugin-debug` and/or `plugin-callback-debug` options when invoking cmake:
+```bash
+cmake -Dplugin-debug=ON -Dplugin-callback-debug=ON ..
+```
+
```

Comment:
It would be nice if this also described (or pointed to a description) of what that gives in Mumble; how do these options help debugging (the plugin) in Mumble?